actions/st2_prep_release_for_st2.sh

```diff
@@ -151,6 +156,48 @@ if [ "${IS_DEV_VERSION}" = "false" ]; then
             fi
         fi
     done
+    for REQUIREMENT_FILE in "${LDAP_REQUIREMENT_FILES[@]}"
+    do
+        echo "Setting ldap branch version in: ${REQUIREMENT_FILE}"
+
+        if [[ ! -e "${REQUIREMENT_FILE}" ]]; then
+            >&2 echo "ERROR: Requirement file ${REQUIREMENT_FILE} does not exist."
+            exit 1
+        fi
+
```

Comment:
The previous `find` command should have returned all of the files, and those files should exist. Is there any reason that the files would be visible/readable by the `find` command but not here?

```suggestion
```